src/components/shipments/DeclareShipments.tsx

```diff
@@ -115,34 +124,40 @@ function DeclareShipments({
   const hasLocalContact = scheduledEvent.localContactId !== null;
 
   return (
-    <Stack spacing={4}>
-      <Typography variant="h6" component="h2">
+    <>
+      <Typography variant="h6" component="h2" sx={{ marginBottom: 3 }}>
         Declare Shipments
       </Typography>
-      <Typography variant="body1">
-        Follow the steps below to declare your shipments:
-        <ol style={{ margin: 0 }}>
-          <li>Add all the shipments (one shipment per parcel)</li>
-          <li>Download labels</li>
-          <li>Post the shipment</li>
-        </ol>
-      </Typography>
-      {!hasLocalContact && (
-        <Alert severity="warning" className={classes.alert}>
-          Shipment declarations are not possible until the local contact has
-          been assigned to your scheduled event
-        </Alert>
-      )}
-
-      <Typography variant="h6">My shipment list</Typography>
-      <QuestionnairesList
-        addButtonLabel="Add Shipment"
-        data={shipments.map(shipmentToListRow) ?? []}
-        onEditClick={onEditClicked}
-        onDeleteClick={onDeleteClicked}
-        onAddNewClick={hasLocalContact ? onAddClicked : undefined}
-        style={{ maxWidth: '100%' }}
-      />
+      <Stack spacing={4} direction="row">
```

Comment:
It is really nice to see whole this magic that can be done with just using Material UI components. 🚀